### ISSUE TITLE
Overwrite files without warning..

### ISSUE THREAD

**Issue by Joepiii** [TeamAmaze/AmazeFileManager#1589]
If I use it as a share handler, like to save as on a WhatsApp profile picture, does it overwrite without warning same named files in the same directory? if I share a WhatsApp profile picture and then save as with amaze into downloads and then do the same for another profile picture, would overwrite the first without warning? if so this is a very serious bug!

**Comment by marcinadd** [comment]
No. You get toast with the message that you cannot overwrite file.

**Comment by bowiechen** [comment]
Closed as I was unable to replicate the issue.

**Comment by VishalNehra** [comment]
I'm able to replicate. It doesn't give toast it overwrites the existing file.
